aries_vcx/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+
+name = "aries-vcx"
+version = "0.1.0"
+authors = ["Absa Group Limited", "Hyperledger Indy Contributors <hyperledger-indy@lists.hyperledger.org>"]
+publish = false
+license = "Apache-2.0"
+edition = "2018"
+
+[lib]
```

Comment:
This section can be removed

Reply:
I woul remove the edition from all packages in the workspace since 2018 is the default. I am not sure about removing the licence, though.